src/screens/StartScreen.tsx

```diff
@@ -1,14 +1,17 @@
 import React from "react";
+import { FunctionComponent } from 'react';
 import { StyleSheet, Text, View, Image, TouchableHighlight } from 'react-native';
+import { HomeScreenProps } from '../navigationTypes';
 
-export const StartScreen = () => {
-    const goRead = () => {}
+export const StartScreen: FunctionComponent<HomeScreenProps> = ({ route, navigation }: HomeScreenProps) => {
+    console.log('in Home screen');
+    const goRead = () => { navigation.navigate('Read'); console.log('switching to Read screen'); }
```

Comment:
I'd say let's add a hardcoded { bookUrl: [that URL to colin's book] } and then make navigationTypes.tsx not have the "| undefined" aspect of that Type

Reply:
Looks good, thanks! I didn't realize we could get rid of initialParams too. nice.